handlers.go

```diff
@@ -252,4 +252,8 @@ func forwardEvent(falcopayload types.FalcoPayload) {
 	if config.WebUI.URL != "" {
 		go webUIClient.WebUIPost(falcopayload)
 	}
+	if config.PolicyReport.Enabled != false {
```

Comment:
```suggestion
	if config.PolicyReport.Enabled == true {
```
It's easier for review to think in a "positive" logic